lib/android/README.md

```diff
@@ -17,10 +17,30 @@ The following are also required for the compilation of the vendored openssl:
 - Perl; and
 - Clang (to be installed with the NDK, which should be included with CMake when if you use Android Studio).
 
-## Build
+## Build and integrate
+
+Build the `.so` libraries for each support ABI.
+
+```sh
+$ make -C ../ ../target/test/android.stamp
+```
+
+Publish the android library to local maven.
 
 ```sh
-$ make -C ../ ../target/test/aar.stamp
+$ ./gradlew :didkit:publishToMavenLocal
+```
```

Comment:
Maybe it would be useful to mention the resulting AAR file, for any non-Maven users?
```suggestion
The AAR file will be built and placed at the following path:
​```
./didkit/build/outputs/aar/didkit-release.aar
​```